### ISSUE TITLE
Loss collapse during training 

### ISSUE THREAD

**Issue by aisagarw** [facebookresearch/simsiam#40]
I am trying to pretrain the SimSiam model on mscoco dataset.. but the loss collapses to -1 very quickly.. What are the possible reasons behind and some suggestions to solve the same?

**Comment by liuzuo-byte** [comment]
我在高光谱图像上训练也是，损失几个epoch就能降到-0.999，分类精度只有55%，为什么呢，我的是单GPU训练，有影响吗
